mmdt/mmdt/settings.py

```diff
@@ -12,25 +12,22 @@
 
 import os
 from pathlib import Path
-
 from decouple import config
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'django-insecure--ld-c!55j+l1gr@alfh&w#+flyd@r*y^wy_r+sokxv&i6d0%tg'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', cast=bool)
+DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
If the default setting for debug is true here, how can we handle it in production? Could you please add a condition here? I believe it will be resolved when we fix this issue: https://github.com/khinezarthwe/mmdt_web_app/issues/65. However, for now, it would be better to add a condition that does not affect production.

Reply:
- [ ] I was added the condition to check it is production or development .Please can you check my code whether it is working properly or not in production.